packages/web/graphql/fragments.ts

```diff
@@ -40,7 +40,7 @@ export const PlayerFragment = gql`
       location
       name
     }
-    daohausMemberships {
+    daohausMemberships @skip(if: $forLoginDisplay) {
```

Comment:
I know that the main reason for adding the `@skip` was because of the overhead associated with pulling the DAOs, but so long as you have it, would you `@skip` everything not necessary for the display in `LoginButton`?

The only fields that need to be returned are the `player_cache/name`, `username` and `player_cache/image`, I think.

Reply:
Ok.
You mean `profile_cache/name`  `profile_cache/image` and `username`. 
Then should i just add @skip directive on field not necessary?